### ISSUE TITLE
Add CI job to generate default Forc manifest and compare against a saved version

### ISSUE THREAD

**Issue by adlerjohn** [FuelLabs/sway#1136]
Currently there are two problems:
1. Examples that are built use a local path for the standard library in order to guarantee they don't break. This mean there are no examples that use the default Forc manifest that uses this repo's `master` branch for the standard library.
1. Such an example is needed for the docs and potentially in other places. https://github.com/FuelLabs/sway/blob/a40472f2a67aa4907c29af0d079ab360c8892b4b/docs/src/introduction/forc_project.md?plain=1#L25

- [ ] Manually generate the default `Forc.toml` and save it to a directory. `examples/` might not be a good directory for this since all examples are built as part of CI currently and modifying that job would be more work.
- [ ] Add a CI job that generates the default `Forc.toml` and compares it against the saved `Forc.toml`. If they differ, fail.
- [ ] Use this saved `Forc.toml` in the book as linked above.

**Comment by mitchmindtree** [comment]
Yeah this sounds reasonable! If we do want to include this under `examples/` it shouldn't be too tricky to filter the example out of the `build_all_examples` Rust script.
